equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/report/Crm5UpperLimitReportControllerTest.java

```diff
@@ -0,0 +1,176 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.controller.report;
+
+import jakarta.validation.ConstraintViolationException;
+import org.assertj.core.api.SoftAssertions;
+import org.assertj.core.api.junit.jupiter.InjectSoftAssertions;
+import org.assertj.core.api.junit.jupiter.SoftAssertionsExtension;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.InvalidDataAccessResourceUsageException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.justice.laa.crime.equinity.historicaldata.exception.DateRangeConstraintViolationException;
+import uk.gov.justice.laa.crime.equinity.historicaldata.exception.ResourceNotFoundException;
+import uk.gov.justice.laa.crime.equinity.historicaldata.service.report.Crm5UpperLimitReportService;
+
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+import static uk.gov.justice.laa.crime.equinity.historicaldata.service.CrmFileService.CRM_TYPE_5;
+
+@SpringBootTest
+@ExtendWith(SoftAssertionsExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ActiveProfiles("local")
+class Crm5UpperLimitReportControllerTest {
+    private List<String> testInvalidFormatDates;
+    private static final String ACCEPTED_PROFILE_TYPES = Integer.toString(CRM_TYPE_5);
+    private static final String DENIED_PROFILE_TYPES = "2,9";
+
+    @InjectSoftAssertions
+    private SoftAssertions softly;
+
+    @Mock
+    Crm5UpperLimitReportService reportService;
```

Comment:
`private` Crm5UpperLimitReportService reportService;

Reply:
Putting private breaks the Mock annotation. Has been like that ever since I started. Happy to give it another try

Reply:
sorted. it worked